crates/symbolicator-service/src/utils/http.rs

```diff
@@ -39,12 +40,55 @@ fn is_external_ip(ip: std::net::IpAddr) -> bool {
     true
 }
 
-pub fn create_client(config: &Config, trusted: bool) -> reqwest::Client {
+/// Various timeouts for all the Downloaders
+#[derive(Copy, Clone, Debug)]
+pub struct DownloadTimeouts {
+    /// The timeout for establishing a connection.
+    pub connect_timeout: Duration,
+    /// The timeout for receiving the first headers.
+    pub head_timeout: Duration,
+    /// An adaptive timeout per 1GB of content.
+    pub streaming_timeout: Duration,
+    /// Global timeout for one download.
+    pub max_download_timeout: Duration,
```

Comment:
Nit: Arguably these could be named without the `_timeout` suffix, but it's fine like this as well.